### PR TITLE
[FLINK-3612] Fix/adjust Table API examples

### PR DESCRIPTION
**Pull request by vasia**
This PR removes the `PageRankTable` example and changes the `WordCountTable` example to work with `Row` type. The `TPCHQuery3Table` example works fine.

### CODE REVIEW

**Comment by vasia** [comment]
I've updated the PR with a change in the `DataSetAggregate` to inject a type conversion mapper if the expected type is not a `Row`. The `WordCountTable` and `JavaTableExample` now work with the custom `WC` type.

File: flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/plan/nodes/dataset/DataSetAggregate.scala [hunk 1]
```diff
@@ -96,24 +93,32 @@ class DataSetAggregate(
...
-      // global aggregation
-      mappedInput.asInstanceOf[DataSet[Row]]
-        .reduceGroup(groupReduceFunction)
-        .returns(rowTypeInfo)
-        .asInstanceOf[DataSet[Any]]
+
+    // if the expected type is not a Row, inject a mapper to convert to the expected type
+    expectedType match {
+      case Some(typeInfo) if typeInfo.getTypeClass != classOf[Row] =>
+        result.map(typeConversion(config, rowTypeInfo, expectedType.get))
```
Comment (fhueske) [inline flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/plan/nodes/dataset/DataSetAggregate.scala]: Add a name for the operator.

**Comment by fhueske** [comment]
Can be merged once the name for the type converter map is added.

**Comment by vasia** [comment]
Thanks! Added the name and merging.

**Pull request closed without merging**